uniconnect-epfl/uniconnect
# feat(network): big updates regarding layout

# What I did
- I replaced our current graph with a new version that has a dynamic simulation
- I removed the description for cards in ContactList
- I added a loading animation for the network screens
- 
# How I did it
- I used d3.js library for simulation

# How to verify it
Given that you have friends on the graph you can double-click, long press, drag the graph and the nodes. You cannot zoom anymore and Animations were removed because unecessary.

To verify it just play along with the graph

# Demo video
https://github.com/uniconnect-epfl/uniconnect/assets/81919237/f7bebb6f-563f-49f5-88df-34ff8049437d

# Pre-merge checklist
The changes I have introduced:
